python/py-cattrs/Portfile

```diff
@@ -4,36 +4,42 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-cattrs
-version             22.2.0
+version             23.2.3
 revision            0
 license             MIT
 maintainers         nomaintainer
 supported_archs     noarch
 platforms           {darwin any}
 
-description         cattrs is an open source Python library for structuring and unstructuring data.
-long_description    {*}${description}. cattrs works best with attrs classes, dataclasses and the usual Python collections, \
-                    but other kinds of classes are supported by manually registering converters.
+description         ${name} is an open-source Python library \
+                    for structuring and unstructuring data.
+long_description    {*}${description}. ${name} works best with attrs classes, \
+                    dataclasses and the usual Python collections, \
+                    but other kinds of classes are supported \
+                    by manually registering converters.
 
 homepage            https://github.com/python-attrs/cattrs
 
-checksums           rmd160  8ae5c1ed14d8f66d7b8445f41cd4abadca6b3c78 \
-                    sha256  f0eed5642399423cf656e7b66ce92cdc5b963ecafd041d1b24d136fdde7acf6d \
-                    size    30050
+checksums           rmd160  900731ad5c47097173e30a23615a48089594302a \
+                    sha256  a934090d95abaa9e911dac357e3a8699e0b4b14f8529bcc7d2b1ad9d51672b9f \
+                    size    610215
 
-python.versions     37 38 39 310 311 312
+python.versions     38 39 310 311 312
 python.pep517       yes
```

Comment:
this is the default for subports 3.7+ and can this be removed

Reply:
Done